modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/vulcan/extension/v1_0/ObjectRelationshipExtensionProvider.java

```diff
@@ -101,15 +102,27 @@ public Map<String, Serializable> getExtendedProperties(
 				ObjectEntryManager objectEntryManager =
 					_objectEntryManagerRegistry.getObjectEntryManager(
 						objectDefinition.getStorageType());
+
+				if (!(objectEntryManager instanceof
```

Comment:
Should we call DefaultObjectEntryManagerProvider here?

Reply:
Yes, I forgot, done [here](https://github.com/liferay-objects/liferay-portal/pull/2092/commits/c6091a925e5569a584745425c005c32c5e43d45a)

Thanks!